src/Network/Network/help/New-AzLoadBalancerBackendAddressPoolTunnelInterface.md

```diff
@@ -0,0 +1,119 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
```

Comment:
The name of the help file should be New-AzLoadBalancerBackendAddressPoolTunnelInterfaceConfig.md

Reply:
thanks!